### ISSUE TITLE
Remove old Lira (U+20A4) symbol from GF Plus?

### ISSUE THREAD

**Issue by RosaWagner** [googlefonts/glyphsets#75]
- Not in use since 2022 in Italy.
- Syria and Libanon occasionally use the Pound sign and not the Lira sign.

Not to confuse with Lira Turkish, this one remains in the set.

**Comment by moyogo** [comment]
TL;DR: Besides compatibility with historical HP Roman encodings and support of misguided character hypercorrection on Wikipedia, there is little use to having LIRA SIGN U+20A4 ₤ in GF Plus as U+00A3 £ is the preferred and largely most common character for lira.

The Unicode charts have the following annotations for U+20A4 ₤:
- intended for lira, but not widely used
- preferred character for lira is 00A3
→ (pound sign - 00A3)
→ (turkish lira sign - 20BA)

The character U+20A4 ₤ seems to have been added to Unicode for compatiblity with some [HP Roman encodings](https://en.wikipedia.org/wiki/HP_Roman).

POUND SIGN U+00A3 £ is the character for the lira in other encodings, and the preferred one in Unicode.
U+00A3 £ can have both shapes with one stroke or with two strokes, even for the British pound.

Beside the odd use on decades old HP devices, in practice LIRA SIGN U+20A4 ₤ was almost never used in Italian even before 2002. POUND SIGN U+00A3 £ was used instead.
U+20A4 ₤ was not in any other code pages or ISO 8859 encodings commonly used in Italian.
Furthermore U+00A3 £ is on standard Italian keyboard layouts (Windows and macOS) while U+20A4 ₤ is not.

Looking at corpuses like [Wortschatz Leipzig Italic corpus](https://wortschatz.uni-leipzig.de/en/download/Italian), U+20A4 ₤ does not appear to have been used in the data gathered in any of the news sentences sets from 2005 to 2020. U+00A3 £ does appear less than 20 times per year.

That said, U+20A4 ₤ does occur many times on the Italian language Wikipedia, most notably in the [Lira italiana](https://it.wikipedia.org/wiki/Lira_italiana) article. This seems to be a misunderstanding of the two characters, U+00A3 £ is still a historically correct and more common symbol for the lira, U+20A4 ₤ was one in HP Roman encodings.
U+20A4 ₤  occurs a total of 370 times in the 2022-09-20 pages-articles dump, or between 1 to 4 times per Wortschatz Wikipedia million sentences for years 2010, 2014, 2016 and 2020.
For comparison, U+00A3 £ (lira, pound or others) occurs 9783 in the same Wikipedia dump and occurs between 98 to 148 times per million sentences for the same Wortschatz sets.

The distinction between the single stroke and the double stroke symbol can be handled at the font level, either by using a font that has one or the other, or a font that has both accesible through typographic features.
This is similar to $ or ¥ which can have one or two strokes depending on the font.
It would be a plus to have the character U+20A4 ₤ and those glyphs variants but that shouldn’t be the norm.

**Comment by RosaWagner** [comment]
Thanks Denis for the documentation! It confirms that U+20A4 should definitely remain in the .nam file, but not necessarily in the glyphset definitions for GF_Latin_Plus — which consist in recommended symbols to draw for designers to complete the basic Latin_Core set.
